cmd/server-test/main.go

```diff
@@ -59,7 +59,6 @@ func main() {
 		successPushJSON := `{"event":"push", "internal_id": "1", "commit_revision":{"base":{"internal_repository_url":"https://github.com/src-d/lookout.git","reference_name":"refs/heads/master","hash":"4eebef102d7979570aadf69ff54ae1ffcca7ce00"},"head":{"internal_repository_url":"https://github.com/src-d/lookout.git","reference_name":"refs/heads/master","hash":"d304499cb2a9cad3ea260f06ad59c1658db4763d"}}}`
 		sendEvent(w, successPushJSON)
 		grepTrue(r, "processing push")
-		grepTrue(r, "comments can belong only to review event but 1 is given")
```

Comment:
hm. We don't have a test for it anymore?
What if an analyzer returns comments for push event? (it can according to api contract)